reqs/README.md

```diff
@@ -9,7 +9,6 @@ Requirements
 - **reqs/requirements-dev-contrib.txt**: Mostly linting tools like `black`, `mypy`, and `ruff`
 - **reqs/requirements-dev-all-contrib-expectations.txt**: The requirements for all of the contrib Expectations; only used in CI when building the Expectation Gallery
 - **reqs/requirements-dev-api-docs-test.txt**: Just `docstring-parser` package needed for generating API docs
-- **reqs/requirements-dev-cloud.txt**: `great_expectations_cloud` and any related deps to get Cloud functional; `pip install ".[cloud]"`
```

Comment:
You are deleting references to this requirements file from here and tasks.py but this file isn't going away. It seems like it should based on this comment. The only file that uses the orjson dependency is [cloud_data_store.py](https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/experimental/metric_repository/cloud_data_store.py). I've asked @anthonyburdi about that file in slack. 

We should delete that file if all references, such as this one, are going away. Otherwise we should update this documentation.

Reply:
thanks - updated!